### ISSUE TITLE
.json extension isn't recognised as JSON language

### ISSUE THREAD

**Issue by olego** [notepad-plus-plus/notepad-plus-plus#2529]
### Description of the Issue
When I open a standalone File.json, the default language is Normal text.
Furthermore, JSON does not appear in StyleConfigurator.

### Steps to Reproduce the Issue
Open File.json.

### Expected Behavior
I expect the language to be JSON.  Or, at least, I want to map .json to JSON.

### Actual Behavior
JSON is one of few languages not appearing in Style Configurator tool window.
By the way, shouldn't all languages in that window be sorted?   "Search Results" is on the bottom.

### Debug Information
Notepad++ v6.9.2
Build time : May 18 2016 - 00:34:05
Path : C:\Program Files (x86)\Notepad++\notepad++.exe
Admin mode : ON
Local Conf mode : OFF
OS : Windows 8.1
Plugins : JSMinNPP.dll mimeTools.dll NppConverter.dll NppTextFX.dll PluginManager.dll VisualStudioLineCopy.Unicode.dll XMLTools.dll

**Comment by dail8859** [comment]
Works fine for me. What theme are you using? You might try deleting your stylers.xml and langs.xml to see if that helps.

**Comment by checor** [comment]
I see you are using v6.9.2. Latest version is v7.1, works fine in it. Are you willing to use the auto-updater to check if the bug remains?

**Comment by olego** [comment]
Yes, installing 7.1 was half of the issue.  The other half was that I had to delete my existing %User%\Dropbox\Notepad++\Langs.xml *and* Stylers.xml.  I've had those files for at least 2 years; now that I've deleted them, they were recreated, and grew 3 times in size.  Is that expected--or should they be auto-merged during a version-to-version upgrade?

(Also, the auto-generated .xml files have a mix of Tabs and Spaces.)
